src/providers/dataProvider/index.ts

```diff
@@ -300,6 +300,8 @@ export const getDataProvider = async (
     {
       resource: constants.R_ITEMS,
       beforeCreate: async (record: CreateResult<Item>) => {
+        record.data.start = new Date(record.data.start).toISOString()
```

Comment:
hello @salony75way - wouldn't it be more effective to change the data generation so that the start and end values are expressed as ISO strings?

Reply:
Hi @IanMayo, Filter was not working with date string so i just changed the date to ISO string format

Reply:
Yes - but let's correct it where we first generate the start/end dates.  Then we won't have to convert it here - or anywhere else.
We _may_ wish to cosmetically `format` dates for display, but it makes sense to store them internally as ISO dates.